velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -583,6 +542,23 @@ TEST_F(ArraySortTest, lambda) {
       "(x, y) -> if(length(x) < length(y), 1, if(length(x) = length(y), 0, -1))");
 }
 
+TEST_F(ArraySortTest, failOnMapTypeSort) {
+  static const StringView kErrorMessage =
```

Comment:
Why StringView? std::string

Reply:
I thought people preferred `StringView` to `std::string` for better performance.

Reply:
I see, the `evaluate` method's param is `std::string`. Fixed.

Reply:
StringView should be used only for data stored in Velox vectors.

std::string_view can be used otherwise. One thing to keep in mind is that neither StringView nor std::string_view own the backing string. Hence, one needs to make sure that the backing string stays alive for as long as the view is alive.

Reply:
Got it, thanks a lot.